Scripts/NuGet/EdFi.Installer.SandboxAdmin/Install-EdFiOdsSandboxAdmin.psm1

```diff
@@ -360,9 +377,15 @@ function New-SqlLogins {
         $odsDbConnectionInfo = (Convert-ConnectionStringtoDatabaseConnectionInfo $Config.MergedSettings.ConnectionStrings.EdFi_Admin)
         $securityDbConnectionInfo = (Convert-ConnectionStringtoDatabaseConnectionInfo $Config.MergedSettings.ConnectionStrings.EdFi_Security)
 
-        Add-SqlLogins $adminDbConnectionInfo $Config.WebApplicationName
-        Add-SqlLogins $odsDbConnectionInfo $WebApplicationName
-        Add-SqlLogins $securityDbConnectionInfo $Config.WebApplicationName
+        if ($Config.IsCustomLogin) { Write-Host ""; Write-Host "Regarding the Admin DB:"; }
```

Comment:
These DB Specific messages were added because otherwise, the same custom username prompt occurs three times and it seems redundant or that maybe an error has occurred because there is no context to say you are answering the custom username prompt for **each** DB